### DISCUSSION TITLE
Nominate @robdimsdale for maintainer role

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#114] [Dependencies Team]
@robdimsdale has expressed interest in becoming a dependencies maintainer, and has made helpful contributions and shared insights in the dependencies area, especially with his work on the new Python dependencies. Rob also has done work and provided feedback on the workflows for the new dependency management system. Overall, I believe Rob would be helpful addition to the Dependencies maintainer team so I would like to nominate him for that role.

https://github.com/orgs/paketo-buildpacks/teams/dependencies-maintainers can I get a 👍 / 👎 ?

**Comment by ryanmoran** [comment]
cc/ @paketo-buildpacks/dependencies-maintainers

**Comment by ryanmoran** [comment]
Congrats @robdimsdale on your Dependencies maintainership! 🎉
